docs/connector-development/testing-connectors/README.md

```diff
@@ -15,8 +19,9 @@ If you want to run the global test suite, exactly like what is run in CI, you sh
 airbyte-ci connectors --name=<connector_name> test
 ```
 
-This will run all the tests for the connector, including the QA checks and the Connector Acceptance tests.
-Connector Acceptance tests require connector configuration to be provided as a `config.json` file in a `.secrets` folder in the connector code directory.
+This will run all the tests that are available for the connector. This can include all of the tests listed above, if we have the appropriate credentials; at a minimum it will include the static asset checks and any tests that exist in a connector's `unit_tests` and `integration_tests` directories.
```

Comment:
```suggestion
CI will run all the tests that are available for a connector. This can include all of the tests listed above, if we have the appropriate credentials. At a minimum, it will include the static asset checks and any tests that exist in a connector's `unit_tests` and `integration_tests` directories.
```

Reply:
do we have any more information on static asset checks?

Reply:
This was just referring to the Connector QA checks. Updated.